### PR TITLE
[Compatible] Update badges and logo

### PR DESCRIPTION
**Pull request by dkijania**
Port https://github.com/MinaProtocol/mina/pull/15889 to compatible

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+| Develop | Compatible | Master | 
+| ------- | ---------- | ---------- |
+| [![Build status - develop](https://badge.buildkite.com/0c47452f3ea619d3217d388e0de522b218db28c3e161887a9a.svg?branch=develop)](https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies) | [![Build status - compatible](https://badge.buildkite.com/0c47452f3ea619d3217d388e0de522b218db28c3e161887a9a.svg?branch=compatible)](https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies) | [![Build status - master](https://badge.buildkite.com/0c47452f3ea619d3217d388e0de522b218db28c3e161887a9a.svg?branch=master)](https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies)
+
+<br />
 
 <a href="https://minaprotocol.com">
-  <img src="https://github.com/MinaProtocol/docs/blob/main/public/static/img/svg/mina-wordmark-redviolet.svg?raw=true&sanitize=true" width="350" alt="Mina logo">
+  <img src="https://minaprotocol.com/wp-content/themes/minaprotocol/img/footerLogo.svg" width="350" alt="Mina logo" />
```
Comment (mrmr1993) [inline README.md:10]: This logo doesn't load for me. Was this change accidental?

**Review by mrmr1993 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (dkijania) [reply README.md:10]: Do you have light or dark theme :)

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: README.md (hunk 1 above)
Comment (dkijania) [reply README.md]: I used different image

**Comment by dkijania** [comment]
!ci-build-me

**Review by SanabriaRusso — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Review by MartinMinkov — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by georgeee** [comment]
PR needs a fix, I'm afraid

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
Apologies for dismissing review but it was a bit stale Matthew is on vacation and this PR is only porting what is done on master and develop, causing some unnecessary conflicts

**Pull request merged**